test/auth.js

```diff
@@ -0,0 +1,22 @@
+export const auth = {
```

Comment:
Worth mentioning that the fixture token expired this morning?

>Thu May 23 2024 10:12:37 GMT+0000

If it's helpful, we use [`luxon`](https://www.npmjs.com/package/luxon) in **forms-designer**

```suggestion
import { DateTime, Duration } from 'luxon'

export const auth = {
```

Reply:
Good idea to check these against `ServerInjectOptions['auth']` types too

Can grab the `@typedef` from the (previously) [much simpler fixture in **forms-designer**](https://github.com/DEFRA/forms-designer/blob/146b33dcdf27c463684ab090def6ff2fbc370caa/designer/server/test/fixtures/auth.js)

```suggestion
/**
 * @satisfies {ServerInjectOptions['auth']}
 */
export const auth = {
```